.github/ISSUE_TEMPLATE/bug_report_linux.yml

```
@@ -0,0 +1,73 @@
+name: Bug Report (Linux)
+description: File a bug report for Linux.
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so that we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: os-version
+    attributes:
+      label: operating system version
+      description: |
+        Which version of your operating sytem do you use?
+
+        On Linux, you should have good results with this command `cat /etc/os-release`
+    validations:
+      required: true
+  - type: textarea
+    id: dangerzone-version
+    attributes:
+      label: Dangerzone version
+      description: Which version of Dangerzone are you using?
+    validations:
+      required: true
+  - type: textarea
+    id: podman-info
+    attributes:
+      label: Podman info
+      description: |
+        If the bug occurs during document conversion, or is otherwise related
+        with Podman, please copy and paste the following commands in your
+        terminal, and provide us with the output:
+
+        ```shell
+        podman version
+        podman info -f 'json'
+        podman images
+        podman run hello-world
+        ```
+
+        This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: textarea
+    id: logs
+    attributes:
+      label: Document conversion logs
+      description: |
+        If the bug occurs during document conversion, we'd like some logs from this process. Please copy and paste the following commands in your terminal, and provide us with the output:
+
+
+        Perform the conversion via the Dangerzone CLI (replace `/path/to/file` with the path to your document):
+
+        ```
+        dangerzone-cli --debug /path/to/file
+        ```
```

```suggestion
        If the bug occurs during document conversion, we'd like some logs from this process. You can perform the conversion via the Dangerzone CLI (replace `/path/to/file` with the path to your document) and copy the output:

        ```
        dangerzone-cli /path/to/file
        ```
```